### DISCUSSION TITLE
Song editor

### DISCUSSION THREAD

**Discussion by iwayori** [Drewol/unnamed-sdvx-clone#494] [Ideas]
Is it possible to add a song editor directly inside the game? The song editor from Kshootmania is pretty old for now and may be hard to understand for new players. I know it's a lot of work to make a song editor inside of the game but it can be a great idea.

**Comment by Drewol** [comment]
This project is not set up in a way that would work well with an in-game editor right now so that would require some major reworks to happen. I have a WIP editor and I think there are a couple others but those projects are either on hold or progressing extremely slowly as there aren't enough contributors with enough time.

My WIP Editor is at https://github.com/Drewol/kson-editor

Related issue: #171
